addons/styled-components/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "tux-addon-styled-components",
+  "version": "0.4.0",
+  "description": "Use Styled Components on the server and client.",
+  "bugs": {
+    "url": "https://github.com/aranja/tux/issues"
+  },
+  "repository": "aranja/tux",
+  "keywords": ["react", "styled-components", "tux-addon"],
+  "main": "lib/index.js",
+  "module": "es/index.js",
+  "types": "lib/index.d.ts",
+  "scripts": {
+    "prepublish": "NODE_ENV=production ../../tasks/build-package.js",
+    "watch": "../../tasks/build-package.js --watch",
+    "test": "../../node_modules/.bin/jest"
+  },
+  "peerDependencies": {
+    "react": ">16.0.0-beta",
+    "react-chain": "^0.5.0"
```

Comment:
Need peer dependency for styled-components. Actually does not depend on react.

Reply:
It also has a dependency on @neutrinojs/compile-loader and peer dependency on neutrino.